lib/meta-storage.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const assert = require('assert');
+
+module.exports = class MetaStorage {
+    constructor(sink) {
+        assert(sink, 'MetaStorage needs a sink');
```

Comment:
Will this surface to the end user? If so, will the end user know what "MetaStorage" is?

Reply:
This is for internal usage as of now.